com.unity.shadergraph/Editor/Generation/TargetResources/Fields.cs

```diff
@@ -14,13 +14,12 @@ internal static class Fields
         public static FieldDescriptor GraphVertex =           new FieldDescriptor(kFeatures, "graphVertex", "FEATURES_GRAPH_VERTEX");
         public static FieldDescriptor GraphPixel =            new FieldDescriptor(kFeatures, "graphPixel", "FEATURES_GRAPH_PIXEL");
         public static FieldDescriptor GraphColorInterp =      new FieldDescriptor(kFeatures, "graphColorInterp", "FEATURES_GRAPH_COLOR_INTERP");
-        public static FieldDescriptor AlphaClip =             new FieldDescriptor(string.Empty, "AlphaClip", "_AlphaClip 1");
```

Comment:
Ah so this was renamed. Will this rename break users?

Reply:
Nope.  Fields are an internal concept in ShaderGraph, not user accessible.